conda_build/render.py

```diff
@@ -69,10 +68,7 @@ def bldpkg_path(m):
 
 def actions_to_pins(actions):
     specs = []
-    if conda_43:
-        spec_name = lambda x: x.dist_name
-    else:
-        spec_name = lambda x: str(x)
+    spec_name = lambda x: x.dist_name
     if 'LINK' in actions:
         specs = [' '.join(spec_name(spec).split()[0].rsplit('-', 2)) for spec in actions['LINK']]
```

Comment:
```suggestion
    if 'LINK' in actions:
        specs = [' '.join(spec.dist_name.split()[0].rsplit('-', 2)) for spec in actions['LINK']]
```

Reply:
yup, I saw a few refactors I could do, just wanted to make this PR easy to review at first, I'll take another pass

Reply:
kk. I also don't mind keeping the cleanup simple -- just looked at the few surrounding lines.
(I'm often unsure when PRs are done/meant for review -- I like the "mark as draft" functionality to communicate that ;).

Reply:
@mbargull no worries, I def try to follow the draft/ready distinction, we haven't been very good at following a consistent process as a community, something to aspire for in 2023

personally, I always appreciate reviews no matter what stage the PR is in